static/game/enemy.js

```diff
@@ -133,11 +134,12 @@ export class Enemy extends Creature {
         addToWorld(childEnemy);
         randDir += (2 * Math.PI) / spawnNum;
       }
-      if (this.matryoshka > 1) {
+      if (this.matryoshka > 1 && this.powerUpTable !== undefined) {
         // drop a level 1 power up as a reward
-        const powerUp = new powerUpTypes[
-          Math.floor(Math.random() * powerUpTypes.length)
-        ](Math.min(5, this.matryoshka - 1), this.pos);
+        const powerUp = new (this.powerUpTable.pick())(
```

Comment:
Does this mean matryoshka enemies /always/ drop powerups?

Reply:
Yes, enemies with matryoshka > 1 _always_ drop power ups. This is the same as the old behavior.